nodejs/src/wrapper.ts

```diff
@@ -0,0 +1,59 @@
+import { diag, DiagConsoleLogger } from '@opentelemetry/api';
+import { BatchSpanProcessor } from '@opentelemetry/sdk-trace-base';
+import { OTLPTraceExporter } from '@opentelemetry/exporter-trace-otlp-http';
+import { NodeTracerProvider } from '@opentelemetry/sdk-trace-node';
+import { registerInstrumentations } from '@opentelemetry/instrumentation';
+import { awsLambdaDetector } from '@opentelemetry/resource-detector-aws';
+import { detectResources, envDetector, processDetector } from '@opentelemetry/resources';
+import { getEnv } from '@opentelemetry/core';
+import { verifyToken } from './utils';
+
+
+const logLevel = getEnv().OTEL_LOG_LEVEL
+
+diag.setLogger(new DiagConsoleLogger(), logLevel)
+
+const ciscoToken = process.env["CISCO_TOKEN"] || '';
+
+if (!ciscoToken) {
+    diag.error('Cisco token was not set');
+}
+
+const { getNodeAutoInstrumentations } = require('@opentelemetry/auto-instrumentations-node');
+
+const instrumentations: any[] =
+    getNodeAutoInstrumentations({
+        '@opentelemetry/instrumentation-aws-lambda': {
+            disableAwsContextPropagation: true
+        },
+    })
+
+async function init() {
+    const detectedResources =
+        await detectResources({
+            detectors: [awsLambdaDetector, envDetector, processDetector]
+        });
+
+    const provider = new NodeTracerProvider({
```

Comment:
I think we will want to add Hexrays propogator as well.
deps:
 "@opentelemetry/id-generator-aws-xray": "^1.0.1",
    "@opentelemetry/sdk-trace-node": "^1.0.1",
    "@opentelemetry/propagator-aws-xray": "^1.0.1",
    "@opentelemetry/propagator-b3": "^1.0.1"

see here how to add propogators: 
https://github.com/aws-observability/aws-otel-lambda/blob/main/nodejs/wrapper-adot/src/adot-extension.ts